spec/services/edge_cache/bust_spec.rb

```diff
@@ -4,6 +4,21 @@
   let(:user) { create(:user) }
   let(:path) { "/#{user.username}" }
 
+  describe "#call" do
+    before do
+      configure_fastly
+    end
+
+    let(:cache_bust_service) { described_class.new(path) }
+
+    it "returns cache bust response" do
+      allow(cache_bust_service).to receive(:bust_fastly_cache).and_return("success")
+
+      cache_bust_service.call
+      expect(cache_bust_service.response).to eq("success")
```

Comment:
I think we ought to move this expectation into the `"when fastly is configured"` context block and add an equivalent expectation to the `"when openresty is configured"` test on line `88` since we're testing the other outcomes of `call` in those tests already. We could also test that `response` is `nil` (since you added logic for that) in the `"when openresty is not configured"` context block.

Just trying to avoid future confusion in this file since it is currently set up to very granularly test fastly and nginx behavior.

Reply:
Good call, I actually solved this issue a different way, wrote the spec, then rewrote the code and left the spec here. Makes much more sense to move it.